core/templates/pages/skill-editor-page/editor-tab/skill-description-editor/skill-description-editor.component.ts

```diff
@@ -34,10 +34,10 @@ export class SkillDescriptionEditorComponent implements OnInit, OnDestroy {
   directiveSubscriptions = new Subscription();
   MAX_CHARS_IN_SKILL_DESCRIPTION = (
     AppConstants.MAX_CHARS_IN_SKILL_DESCRIPTION);
-  skillRights: SkillRights;
-  skill: Skill;
-  skillDescriptionEditorIsShown: boolean;
-  tmpSkillDescription: string;
+  skillRights: SkillRights = null;
+  skill: Skill = null;
+  skillDescriptionEditorIsShown: boolean = null;
+  tmpSkillDescription: string = null;
```

Comment:
Are these changes expected in a fe tests PR?

Reply:
Currently, these values are undefined initially.
In order to explicitly defined them as empty values, I've used null.
(I will revert if these changes are not appropriate)

Reply:
Currently, these values are undefined initially.
In order to explicitly defined them as empty values, I've used null.
(I will revert if these changes are not appropriate)

Reply:
I think this is fine. Thanks for clarifying.